blocks/init/src/Blocks/custom/tab/tab-block.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { getUnique, overrideInnerBlockSimpleWrapperAttributes } from '@eightshift/frontend-libs/scripts/editor';
+import { checkAttr, getAttrKey } from '@eightshift/frontend-libs/scripts/helpers';
+import { InspectorControls } from '@wordpress/block-editor';
+import { TabEditor } from './components/tab-editor';
+import { TabOptions } from './components/tab-options';
+import { useSelect } from '@wordpress/data';
+import manifest from './manifest.json';
+
+export const Tab = (props) => {
+	const {attributes, clientId, setAttributes} = props;
```

Comment:
```suggestion
	const {
		attributes,
		clientId,
		setAttributes
	} = props;
```

Just a bit more dev-friendly formatting.